### ISSUE TITLE
Train your model: LightGBM

### ISSUE THREAD

**Issue by MarynaSD** [Bootcamp-IA-P4/Proyecto-Clasificaci-n_grupo6#19]
Train your model: LightGBM
Evaluate model performance using the following metrics:
Accuracy
Precision
Recall
F1-score
Build a confusion matrix for each model.

**Comment by fergarcat** [comment]
[LightGBM] [Warning] lambda_l1 is set=6.663357419343838, reg_alpha=0.0 will be ignored. Current value: lambda_l1=6.663357419343838
[LightGBM] [Warning] lambda_l2 is set=0.03707103328485642, reg_lambda=0.0 will be ignored. Current value: lambda_l2=0.03707103328485642

Evaluación en test:
              precision    recall  f1-score   support

           0       0.95      0.92      0.93      2059
           1       0.63      0.72      0.67       382

    accuracy                           0.89      2441
   macro avg       0.79      0.82      0.80      2441
weighted avg       0.90      0.89      0.89      2441

Confusion Matrix:
 [[1896  163]
 [ 106  276]]
ROC-AUC: 0.9234639648688301
F1-score entrenamiento: 0.6948
F1-score test:          0.6724
Overfitting (train - test): 0.0225

**Comment by fergarcat** [comment]
Feature Importance Chart
![Image](https://github.com/user-attachments/assets/902a505b-7375-491c-91eb-79a90771b041)
